### PR TITLE
mining / stacking improvements

### PR DESCRIPTION
**Pull request by ice-orion**
work in progress

### CODE REVIEW

File: src/constants/links.ts [hunk 1]
```diff
@@ -1,4 +1,11 @@
...
-export const APP_STORE_LINK = '';
-export const PLAY_STORE_LINK = '';
+export const LINKS = {
+  //TODO: set store links
+  APP_STORE: '',
+  PLAY_STORE: '',
+  PRE_STAKING: 'https://ice.io/pre-staking/',
+  PRIVACY: 'https://ice.io/privacy-policy/',
+  TERMS: 'https://ice.io/terms/',
+  BLOCK_EXPLORER: 'https://explorer.ice.io/', //TODO: use when https://github.com/ice-blockchain/mobile-app/pull/100 is merged
```
Comment (ice-ares) [inline src/constants/links.ts:10]: its per environment!

File: src/translations/translations.json [hunk 2]
```diff
@@ -435,14 +435,14 @@
...
-            "benefits_description": "Pre-stake your mining rewards for up to five years and increase your mining rate by up to 500%",
+            "benefits_description": "Pre-stake your mining rewards for up to 5 years and increase your mining rate by up to 250%",
             "bonus_label": "Pre-Staking Bonus",
             "period_label": "Staked Period",
             "title": "Pre-Staking",
             "current_rate": "Current Rate",
             "allocation": "Allocation",
             "description": "Staking is a process that involves committing your crypto assets to support a blockchain network and confirm transactions",
-            "terms_agree": "By clicking Pre-Stake Now, you agree to the",
+            "terms_agree": "I agree to the",
```
Comment (ice-ares) [inline src/translations/translations.json:445]: don't fragment the translation! add the whole text into a key, and add parameters

File: src/constants/links.ts [hunk 3]
```diff
@@ -1,4 +1,11 @@
 // SPDX-License-Identifier: BUSL-1.1
 
-export const APP_STORE_LINK = '';
-export const PLAY_STORE_LINK = '';
+export const LINKS = {
+  //TODO: set store links
+  APP_STORE: '',
+  PLAY_STORE: '',
+  PRE_STAKING: 'https://ice.io/pre-staking/',
```
Comment (ice-ares) [inline src/constants/links.ts:7]: those links might have different versions for each language

File: src/constants/links.ts (hunk 1 above)
Comment (ice-orion) [reply src/constants/links.ts:10]: `https://staging.explorer.ice.io/`
and 
`https://explorer.ice.io/`
?

**Pull request closed without merging**